### PR TITLE
finding least postive number python 

### PR DESCRIPTION
**Pull request by devanshi-katyal**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?
yes
(Write your answer here.)

### Description
Added the python solution for finding least positive number.
I am sorry if i am sending the wrong pull request
(Write your answer here.)

### Checklist

- [X ] I've read the contribution guidelines.
- [X ] I've checked the issue list before deciding what to submit.
- [X ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)
https://github.com/TesseractCoding/NeoAlgo/issues/855

### CODE REVIEW

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@devanshi-katyal add more description of the program. Add space and time complexity. Move this program to `cp` sections and update the filename with a suitable title.

**Comment by devanshi-katyal** [comment]
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?
yes
### Description
Added the python solution for finding least positive number.
space complexity:O(1)
time complexity:O(n)
### Checklist

- [X ] I've read the contribution guidelines.
- [X ] I've checked the issue list before deciding what to submit.
- [X ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)
https://github.com/TesseractCoding/NeoAlgo/issues/855

**Comment by devanshi-katyal** [comment]
> @devanshi-katyal add more description of the program. Add space and time complexity. Move this program to `cp` sections and update the filename with a suitable title.

i have made the changes.please check.

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@devanshi-katyal add the description in  `findingLeastPositiveNumber.py` program in the form of comments and move this program to ` Python/cp` not in ` Python/ds`.

**Comment by devanshi-katyal** [comment]
made the changes

**Review by Abhijit2505 — APPROVED** [review]
Looking Good To Me

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@devanshi-katyal Please check the file name its `findingLeastPositiveNumber.py.py` change to `findingLeastPositiveNumber.py`. Update the README. Your code is not following PEP 8 rules. You can check your program here http://pep8online.com/.

**Comment by devanshi-katyal** [comment]
hi, i made the changes but i am not sure how to update the read me.
please help me with that

**Comment by iamrajiv** [comment]
@devanshi-katyal `NeoAlgo/Python/README.md` by adding the program in the list.

**Comment by devanshi-katyal** [comment]
done! please have a look. i am sorry but you tagged the wrong person in the previous comment.

**Review by iamrajiv — APPROVED** [review]

File: Python/cp/findingLeastPositiveNumber.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+def findpositive(arr, size):
+    j = 0
+    for i in range(size):
+        if (arr[i] <= 0):
+            arr[i], arr[j] = arr[j], arr[i]
+            j += 1
+    return MainFunction(arr[j:], size - j)
+arr = list(map(int, input().split(" ")))
+print("the smallest missing number", findpositive(arr, len(arr)))
```
Comment (atarax665) [inline Python/cp/findingLeastPositiveNumber.py]: Add sample input/output in commented form.

**Review by atarax665 — CHANGES_REQUESTED** [review]

**Comment by devanshi-katyal** [comment]
please merge this pl
thanks

**Comment by devanshi-katyal** [comment]
Please merge this pull request.

**Comment by iamrajiv** [comment]
@all-contributors please add @devanshi-katyal for code

**Pull request merged**